### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by thaisdMM**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by thaisdMM — COMMENTED** [review]
I believe that in line 36 it would be better if the person had the option to not add a friend as a reviewer if they don't have one. If their purpose on GitHub is to study, they may not have anyone to collaborate with.

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (thaisdMM) [inline index.html:51]: <h2 hidden> Let's Go! You can do it! 💪🏼 </h2>

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
```
Comment (thaisdMM) [inline index.html:50]: <h2 hidden> Game over. Don't give up! 💪🏼  You can do it! Refresh to play again 🧑‍💻 🤖</h2>

**Comment by andremedeiros13** [comment]
Ola @thaisdMM 
Esta correto a alteração

**Pull request merged**